src/Appwrite/Specification/Format/OpenAPI3.php

```diff
@@ -120,18 +120,10 @@ public function parse(): array
         foreach ($this->routes as $route) {
             $url = \str_replace('/v1', '', $route->getPath());
             $scope = $route->getLabel('scope', '');
-            $hide = $route->getLabel('sdk.hide', false);
             $consumes = [$route->getLabel('sdk.request.type', 'application/json')];
 
-            if ($hide) {
-                continue;
-            }
-
-            $method = $route->getLabel('sdk.method', [\uniqid()]);
-            if (\is_array($method)) {
-                $method = $method[0];
-            }
 
+            $id = $route->getLabel('sdk.method', \uniqid());
```

Comment:
Let's revert this back to `$method` and keep using it throughout, same for `Swagger2`

Reply:
Done